.gitignore

```diff
@@ -6,6 +6,7 @@ test/unit/coverage
 test/e2e/reports
 selenium-debug.log
 *.swp
+sendgrid.env
```

Comment:
is this necessary?

Reply:
I don't think so. I don't think that file is even needed because it was just for setting the API key which is already set in the config file.